client/settings/settings-manager/style.scss

```diff
@@ -0,0 +1,81 @@
+/* this can be removed after all the settings sections are converted to the new layout */
```

Comment:
It looks like we're updating some styles relating to Section Layouts (e.g. `.wcpay-settings-layout`) and Section Settings (e.g. `.settings-section__details`) here.

In my opinion, we should apply those styles to the specific component style files ([SettingsLayout](https://github.com/Automattic/woocommerce-payments/blob/e8068180d9814daf3a58d19ac94d4c5e3dcfba9e/client/settings/settings-layout/style.scss#L1) and [SettingsSection](https://github.com/Automattic/woocommerce-payments/blob/cc6fa05c83cf38833c2e79bb8ee9af7be6c75659/client/settings/settings-section.scss#L1)) because otherwise, if future updates are made to those files, it can cause conflicts with these changes and may be confusing to know where the styles need to be updated.

Reply:
thanks for the suggestion @dmallory42 . Indeed, the best place for the styles added is where you mentioned, but since we need it only for Payment Settings screen, I added all changes to the settings manager. Adding it in other way, I think the styles will apply to all sections ( eg. multicurrency, Apple customize page etc ). 
When all sections will be re-designed, I think we can move the styles in proper places.